armory/datasets/preprocessing.py

```diff
@@ -57,6 +58,55 @@ def carla_over_obj_det_dev(element, modality="rgb"):
     )
 
 
+def carla_video_tracking_preprocess(element, max_frames, split):
```

Comment:
I'd suggest having `max_frames` be a kwarg that defaults to None

Reply:
It defaults to none in the registered functions:
https://github.com/twosixlabs/armory/blob/742d6cb0fcf3fcdc00242172668ad5652c81486c/armory/datasets/preprocessing.py#L106
https://github.com/twosixlabs/armory/blob/742d6cb0fcf3fcdc00242172668ad5652c81486c/armory/datasets/preprocessing.py#L117

Reply:
I'd have it also be a kwarg that defaults to None in the functions the registered one is calling, this way you're not forced to pass max_frames if you're using one of those functions directly and not via the registered one

Reply:
Addressed in 9f71947cee22b7f5c941312dabe21acf37418ebc.